src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -26,6 +26,8 @@ public static class RpcWorkerConstants
 
         // Section names in host.json or AppSettings
         public const string LanguageWorkersSectionName = "languageWorkers";
+        public const string WorkerProbingPathsSectionName = "probingPaths";
```

Comment:
Do we want this to be "workerPobingPaths"? It's not clear what the probing path is for otherwise, unless this is a subsection under "workers" or something like that?